src/@primer/gatsby-theme-doctocat/nav.yml

```
@@ -56,6 +56,33 @@
       url: /foundations/color
     - title: Responsive
       url: /foundations/responsive
+    - title: CSS Utilities
```

For consistency, let's use sentence case here:
```suggestion
    - title: CSS utilities
```